documentation/modules/snip-migrating-luns.adoc

```diff
@@ -0,0 +1,11 @@
+:_content-type: SNIPPET
+
+{project-short} supports direct migration of virtual machines with logical unit numbers (LUNs), other than Fibre Channel, from {rhv-full} ({rhv-short}) source providers as follows:
+
+* Unlike disk images that _are copied_ from a source provider to a target provider, LUNs are detached, but not removed, from virtual machines in the source provider and then attached to the virtual machines (VMs) that are created in the target provider.
+
+* LUNs are not removed from the source provider during the migration in case fallback to the source provider is required. However, before re-attaching the LUNs to VMs in the source provider, be sure that the LUNs are not used by VMs on the target environment at the same time, which might lead to data corruption.
+
+* If you are migrating a virtual machine with a direct LUN disk, ensure that the nodes in the {virt} destination cluster that the VM is expected to run on can access the backend storage.
```

Comment:
as we place this part in the prerequisites for RHV provider, which I believe is the right place for it, maybe we should start with this point as a prerequisite and then elaborate on it with the rest of the text here?